demo/build.gradle

```diff
@@ -14,8 +14,8 @@ android {
 
     lintOptions {
         lintConfig teamPropsFile('static-analysis/lint-config.xml')
-        abortOnError true
-        warningsAsErrors true
+        abortOnError false
+        warningsAsErrors false
```

Comment:
do you need to specify these? Wouldn't they be false by default?

Reply:
Maybe?It might be nice to be explicit though?  
Docs don't say if it is 🤷‍♂️ 
http://google.github.io/android-gradle-dsl/current/com.android.build.gradle.internal.dsl.LintOptions.html#com.android.build.gradle.internal.dsl.LintOptions

Reply:
🤷‍♂️ keep them then 😄